source/js/Directions.js

```diff
@@ -20,31 +124,114 @@ class Directions extends HTMLElement {
     const directionList = searchForKey(data, "recipeInstructions");
 
     const wrapper = document.createElement("article");
-    wrapper.classList.add("example");
+    wrapper.classList.add("background");
 
     const directions = document.createElement("p");
     directions.classList.add("directions");
     directions.textContent = "Directions";
+    directions.classList.add("header");
     wrapper.appendChild(directions);
 
+    // let button_wrapper = document.createElement('button_wrapper');
+    // button_wrapper.classList.add('buttonstyle');
+
     // Parsing data to create the direction list.
     const list = document.createElement("ol");
+    list.classList.add("olStyle");
     for (let i = 0; i < directionList.length; i++) {
-      const listItem = document.createElement("li");
-      listItem.textContent = `${directionList[i].name}`;
-      list.appendChild(listItem);
-
+      let direction = document.createElement('div');
+      let x = i+1;
+      let text_button = document.createElement('button');
+      let wrapper = document.createElement('div');
+      text_button.appendChild(wrapper);
+      let cb = document.createElement('input');
+      cb.type = 'checkbox';
+      cb.classList.add('checkbox');
+      direction.appendChild(cb);
+      direction.appendChild(text_button);
+      direction.classList.add('direction');
+      wrapper.classList.add('buttonstyle');
+      let text = document.createElement('p');
+      let down_arrow = document.createElement('img');
+      down_arrow.setAttribute('src', './images/arrow-down.png');
+      let up_arrow = document.createElement('img');
+      up_arrow.setAttribute('src', './images/arrowUp.png');
+      down_arrow.classList.add('downArrow');
+      up_arrow.classList.add('upArrow');
+      
+      text.textContent = `${x +')'+ ' ' + directionList[i].name}`;
+      text.classList.add('text');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/cse110-fa21-group27/cse110-fa21-group27/pullRequest?prid=8460933)
```suggestion
      text.classList.add("text");
```